airflow/providers/amazon/aws/sensors/dynamodb.py

```diff
@@ -0,0 +1,96 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+import boto3
+from pprint import pprint
+from botocore.exceptions import ClientError
+
+from airflow.sensors.base import BaseSensorOperator
+from typing import TYPE_CHECKING, Optional
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class DynamoDBSensor(BaseSensorOperator):
+    """
+    Waits for an attribute value to be present for an item in a DynamoDB table.
```

Comment:
not super familiar with DynamoDB, but would there possibly be more things to sense from airflow in the future ? In which case, maybe we could find a more specific name than the very broad `DynamoDBSensor` ?

Reply:
`DynamoDBItemAttributeValueSensor` would be the most specific name possible for this class. We could shorten it to `DynamoDBValueSensor`?

Reply:
yeah good for me 👍

Reply:
Changed to `DynamoDBValueSensor`.